modules/ui/fields/input.js

```diff
@@ -34,11 +34,41 @@ export function uiFieldText(field, context) {
             .catch(function() { /* ignore */ });
     }
 
-    function i(selection) {
-        var entity = _entityIDs.length && context.hasEntity(_entityIDs[0]);
-        var preset = entity && presetManager.match(entity, context.graph());
-        var isLocked = preset && preset.suggestion && field.id === 'brand';
+
+    function calcLocked() {
+        // Protect certain fields that have a companion `*:wikidata` value
+        var isLocked = (field.id === 'brand' || field.id === 'network' || field.id === 'operator' || field.id === 'flag') &&
+            _entityIDs.length &&
+            _entityIDs.some(function(entityID) {
+                var entity = context.graph().hasEntity(entityID);
+                if (!entity) return false;
+
+                var which = field.id;   // 'brand', 'network', 'operator', 'flag'
+
+                // If the value was already edited manually then unlock and allow further editing
+                var base = context.graph().base().entities[_entityIDs[0]];
+                if (base) {
+                    var hasOriginalValue = entity.tags[which] && entity.tags[which] === base.tags[which];
+                    if (!hasOriginalValue) return false;
+                }
+
+                // Features linked to Wikidata are likely important and should be protected
+                if (entity.tags.wikidata) return true;
+
+                var preset = presetManager.match(entity, context.graph());
+                var isSuggestion = preset && preset.suggestion;
+
+                // Lock the field if there is a value and a companion `*:wikidata` value
+                return isSuggestion && !!entity.tags[which] && !!entity.tags[which + ':wikidata'];
```

Comment:
`flag:name` goes with `flag:wikidata` rather than `flag:name:wikidata`. (Sorry, I didn’t think of the inconsistency when promoting `flag:name` instead of `flag`.)

Maybe this is a happy coincidence, since the mapper may need to come up with the flag name in the local language anyways. Also, for some kinds of flags, it’s common enough for the same flag to represent a different entity. (The example that comes to mind is the flag of the Vatican City, which is also a religious flag representing the Holy See, but there are also plenty of examples among historical flags.)

Reply:
> Maybe this is a happy accident, since the mapper may need to come up with the flag name in the local language anyways.

Yes it is 👍   It means this code would lock a `flag` field, but not a `flag:name` field, which is what I think you want..